features/createroom/impl/src/main/kotlin/io/element/android/features/createroom/impl/root/CreateRoomRootEvents.kt

```diff
@@ -16,7 +16,11 @@
 
 package io.element.android.features.createroom.impl.root
 
+import io.element.android.libraries.matrix.ui.model.MatrixUser
```

Comment:
A side note (not for you then) but we should probably put the MatrixUser inside the `matrix` module instead of `matrix.ui`, I find it weird to be seperated.